blocks/init/src/Blocks/components/social-links/social-links.php

```diff
@@ -40,7 +41,7 @@
 		?>
 		<li class="<?php echo \esc_html("{$componentClass}__item"); ?>">
 			<a class="<?php echo \esc_html("{$componentClass}__link"); ?>" href="<?php echo esc_url($href); ?>" title="<?php echo esc_attr($title); ?>" target="_blank" rel="nofollow noopener">
-				<?php echo \wp_kses_post($icons[$icon]); ?>
+				<?php echo \wp_kses_post($manifest['icons'][$icon]); ?>
```

Comment:
No need to escape it here (might break some icons) right?. If so please remove wp_kses_post and add phpcs:ignore